content/6_Advanced/String_Search.mdx

```diff
@@ -595,18 +595,6 @@ int main() {
 
 <Problems problems="aho" />
 
-<IncompleteSection>
+# Practice Problems
```

Comment:
are these problems for the entire module? 

also just leave it as problems, not practice problems for consistency. you can just fix this when u add the other problem :)

Reply:
Sure!